lib/coverage.js

```diff
@@ -133,9 +133,9 @@ const Coverage = module.exports = inherit({
             return;
         }
 
-        var base64Prefix = 'data:application/json;base64,';
-        if (_.includes(sourceMapUrl, base64Prefix)) {
-            var base64Str = sourceMapUrl.split(base64Prefix)[1],
+        var base64Prefix = /^data:application\/json;(charset=utf-8;)?base64,/;
```

Comment:
we don't need to use capturing brackets here:
`var base64Prefix = /^data:application\/json;(?:charset=utf-8;)?base64,/;`